engine/gamesys/src/gamesys/scripts/script_resource.cpp

```diff
@@ -1251,13 +1252,15 @@ static int ReleaseResource(lua_State* L)
 {
     DM_LUA_STACK_CHECK(L, 0);
     dmhash_t path_hash                      = dmScript::CheckHashOrString(L, 1);
+    HResourceDescriptor rd = dmResource::FindByHash(g_ResourceModule.m_Factory, path_hash);
+    if (!rd) {
+        luaL_error(L, "Could not release resource: %s", dmHashReverseSafe64(path_hash));
+        return 0;
```

Comment:
it should be "return luaL_error()" (it's the recommended way from Lua)

Reply:
👍